### PR TITLE
S03 create Admin Interface including Dashboard, tags crud, videos crud & users crud

### PR DESCRIPTION
**Pull request by maxdlr**
- add admin dashboard shell
- add video & tag CRUDs to Admin Dashboard
- add symfony UX Chart.js for admin dashboard purposes
- add functional & customizable admin dashboard
- doing update 'new' button on videoCrudController
- doing upload video button as easyadmin templates
- add upload video button & User's admin crud

### CODE REVIEW

**Review by l00ma — APPROVED** [review]

File: src/Controller/Admin/DashboardController.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        $chartManager = $this->chartManager;
+    }
+
+    #[Route('/admin', name: 'admin_dashboard')]
+    public function index(): Response
+    {
+        $tags = $this->tagRepository->findAll();
+        $videos = $this->videoRepository->findAll();
+
+        // --------------------------------------------------------
```
Comment (aldaitzwild) [inline src/Controller/Admin/DashboardController.php:34]: What's the point of those comments ?

File: src/Controller/Admin/TagCrudController.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    // In the future, this could be used when 'game styles' will be implemented.
+    /*
+    public function configureFilters(Filters $filters): Filters
+    {
+        return $filters
+            ->add(EntityFilter::new('video'))
+        ;
+    }
+    */
```
Comment (aldaitzwild) [inline src/Controller/Admin/TagCrudController.php:42]: Dead code

File: src/Service/ChartManager.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+{
+    public function __construct(
+        private ChartBuilderInterface $chartBuilder,
+    ) {
+    }
+
+    public function createBarChartBy(
+        array $labels = [],
+        array $data = [],
+        string $title = 'Change Me',
```
Comment (aldaitzwild) [inline src/Service/ChartManager.php:19]: Yes I guess you should ...

File: src/Service/ChartManager.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Service;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\UX\Chartjs\Builder\ChartBuilderInterface;
+use Symfony\UX\Chartjs\Model\Chart;
+
+class ChartManager extends AbstractController
```
Comment (aldaitzwild) [inline src/Service/ChartManager.php:9]: No a Service shouldn't be an AbstractController ...

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Controller/Admin/VideoCrudController.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Filters;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\AssociationField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\DateTimeField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\EmailField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextareaField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
+use EasyCorp\Bundle\EasyAdminBundle\Filter\EntityFilter;
```
Comment (aldaitzwild) [inline src/Controller/Admin/VideoCrudController.php:17]: Please clean thoses uses (only the one you don't actually use)

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**